src/command_modules/azure-cli-vm/tests/test_vm_commands.py

```diff
@@ -413,7 +413,7 @@ def body(self):
         # Expecting no results
         self.cmd('vm list --resource-group {}'.format(self.resource_group), checks=NoneCheck())
         self.cmd('vm create --resource-group {0} --location {1} --name {2} --admin-username ubuntu '
-                 '--image Canonical:UbuntuServer:14.04.4-LTS:latest --admin-password testPassword0 '
+                 '--image UbuntuLTS --admin-password testPassword0 '
```

Comment:
We should add a scenario test to make sure we can actually successfully provision a plan-based VM with this change.